### PR TITLE
[404114] Change message texts for configuration/unconfiguration of Xtext

### PR DESCRIPTION
**Pull request by kthoms**

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=404114

File: plugins/org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/nature/messages.properties [hunk 1]
```diff
@@ -1,3 +1,3 @@
-NatureAddingEditorCallback_MessageDialog_Title=Add Xtext Nature
-NatureAddingEditorCallback_MessageDialog_Msg0=Do you want to add the Xtext nature to the project '
+NatureAddingEditorCallback_MessageDialog_Title=Configure Xtext
+NatureAddingEditorCallback_MessageDialog_Msg0=Do you enable Xtext for the project '
```
Comment (spoenemann) [inline plugins/org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/nature/messages.properties]: Incomplete sentence: should be
```
Do you want to enable Xtext for the project ...
```
or, in order to be more consistent with the terms used in the context menu,
```
Do you want to convert ... to an Xtext project?
```

**Comment by dhuebner** [comment]
It's a very good idea, our menu entry looks a bit alien:
bildschirmfoto 2016-04-07 um 16 50 10

File: plugins/org.eclipse.xtext.ui.shared/plugin.properties [hunk 2]
```diff
@@ -2,7 +2,7 @@ OpenModelElementCommand.name=Open Model Element
 OpenModelElementCommand.description=Open the declaration of an Xtext element
 XtextBuilder.name=Xtext Project Builder
 XtextNature.name=Xtext Project Nature
-XtextNatureAddAction.label=Add Xtext Nature
-XtextNatureRemoveAction.label=Remove Xtext Nature
+XtextNatureAddAction.label=Convert to Xtext project
```
Comment (svenefftinge) [inline plugins/org.eclipse.xtext.ui.shared/plugin.properties]: uppercase P

File: plugins/org.eclipse.xtext.ui.shared/plugin.properties [hunk 3]
```diff
@@ -2,7 +2,7 @@ OpenModelElementCommand.name=Open Model Element
...
+XtextNatureRemoveAction.label=Unconfigure Xtext
```
Comment (svenefftinge) [inline plugins/org.eclipse.xtext.ui.shared/plugin.properties]: Should we remove this all together? Other plugins doesn't seem to support removing as well.

File: plugins/org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/nature/messages.properties [hunk 4]
```diff
@@ -1,3 +1,3 @@
-NatureAddingEditorCallback_MessageDialog_Title=Add Xtext Nature
-NatureAddingEditorCallback_MessageDialog_Msg0=Do you want to add the Xtext nature to the project '
+NatureAddingEditorCallback_MessageDialog_Title=Configure Xtext
```
Comment (svenefftinge) [inline plugins/org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/nature/messages.properties]: I think this should use the same lingo than the configure action, i.e. "Convert to Xtext Project" and Miro's suggestion.

File: plugins/org.eclipse.xtext.ui.shared/plugin.properties (hunk 3 above)
Comment (kthoms) [inline plugins/org.eclipse.xtext.ui.shared/plugin.properties]: Maybe makes sense. It is unlikely that someone disables Xtext once it was configured. But if someone does want to do this, how should it be done? I think it is too prominent to have this option always in the "Configure" menu. How about having this in the project preference page? If so, shall I open a separate bug?

File: plugins/org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/nature/messages.properties [hunk 5]
```diff
@@ -1,3 +1,3 @@
-NatureAddingEditorCallback_MessageDialog_Title=Add Xtext Nature
-NatureAddingEditorCallback_MessageDialog_Msg0=Do you want to add the Xtext nature to the project '
-NatureAddingEditorCallback_MessageDialog_Msg1='?
+NatureAddingEditorCallback_MessageDialog_Title=Configure Xtext
+NatureAddingEditorCallback_MessageDialog_Msg0=Do you want to convert '
+NatureAddingEditorCallback_MessageDialog_Msg1='to an Xtext project?
```
Comment (spoenemann) [inline plugins/org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/nature/messages.properties]: Cosmetic: I think there should be a space between `'` and `to`.

**Pull request merged**